### ISSUE TITLE
Offlining command processor doesn't deactivate command burst immediately

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#1761]
```
[Bifrost, Solo]

Domination Ballistic Control System
Domination Ballistic Control System
Nanofiber Internal Structure II

Gistii A-Type 5MN Microwarpdrive
Medium 'Canyon' Shield Extender
'Delineative' Warp Scrambler
Domination Stasis Webifier
Shadow Serpentis Warp Disruptor

Skirmish Command Burst II, Interdiction Maneuvers Charge
Skirmish Command Burst II, Rapid Deployment Charge
Dread Guristas Light Missile Launcher, Caldari Navy Inferno Light Missile
Dread Guristas Light Missile Launcher, Caldari Navy Inferno Light Missile
Dread Guristas Light Missile Launcher, Caldari Navy Inferno Light Missile
Dread Guristas Light Missile Launcher, Caldari Navy Inferno Light Missile

Small Processor Overclocking Unit I
Small Command Processor I
```
Current release 2.5 branch.

Steps:
1) Offline command processor

Expected: one of command bursts is deactivated
Actual: command burst is not deactivated. It is actually deactivated next time you do anything (e.g. toggle another module, or even online command processor back)

**Comment by djrscally** [comment]
This seems to be fixed in v2.9.4

**Comment by DarkFenX** [comment]
True, closing it. I think it has been fixed earlier with pretty significant command rework in one of 2.9.x releases.
